PWGHF/TableProducer/HFTrackIndexSkimsCreator.cxx

```diff
@@ -336,14 +338,14 @@ struct HfTagSelTracks {
       MY_DEBUG_MSG(isProtonFromLc, LOG(info) << "\nWe found the proton " << indexBach);
 
       int statusProng = BIT(CandidateType::NCandidateTypes) - 1; // selection flag , all bits on
-      //bool cutStatus[CandidateType::NCandidateTypes][nCuts];
-      //if (debug) {
-      //  for (int iCandType = 0; iCandType < CandidateType::NCandidateTypes; iCandType++) {
-      //    for (int iCut = 0; iCut < nCuts; iCut++) {
-      //      cutStatus[iCandType][iCut] = true;
-      //    }
-      //  }
-      //}
+      // bool cutStatus[CandidateType::NCandidateTypes][nCuts];
+      // if (debug) {
+      //   for (int iCandType = 0; iCandType < CandidateType::NCandidateTypes; iCandType++) {
+      //     for (int iCut = 0; iCut < nCuts; iCut++) {
+      //       cutStatus[iCandType][iCut] = true;
+      //     }
+      //   }
+      // }
```

Comment:
Where are all these formatting changes coming from? I don't think that clang has a problem with the current version.

Reply:
Actually I don't know: I also thought that the previous version was already fine, but at the end those changes are automatically suggested and implemented when running clang as suggested in the coding conventions, so I left them